test/unit/spec/narrative_core/upload/stagingAreaViewer-spec.js

```diff
@@ -229,5 +229,35 @@ define ([
             expect(dlNode.getAttribute('src')).toEqual('some_url');
         });
 
+        it('should properly render the import as dropdown', async () => {            
+            let file = {
+                name: 'render_file_test.fa',
+                path: fakeUser + '/render_file_test.fa',
+                mtime: 1532738637555,
+                size: 49233,
+                source: 'KBase upload',
+                isFolder: false,
+                imported: {}
+            };
+
+            await stagingViewer.render();
+            stagingViewer.renderFiles([file]);
+            let placeholder = $targetNode.find('span.select2-selection__placeholder').html();
+            expect(placeholder).toContain('Select a type');
+
+            let options = $targetNode.find('.select2-hidden-accessible').html();
+            expect(options).toContain('FASTQ Reads');
+            expect(options).toContain('SRA Reads');
+            expect(options).toContain('GenBank Genome');
```

Comment:
You could also put all your strings in an array and then run `expect(options).toContain(string)` for each string in the array (I usually do that for anything with a repeated test).